setup.py

```diff
@@ -22,16 +23,21 @@
 # $ pip install wheel
 # $ python setup.py bdist_wheel --universal
 # $ twine upload dist/*
-
+requirements = ['Flask', 'Pillow', 'requests', 'flask-cors',
+                'psutil', 'jsonschema', 'onnx-mxnet>=0.4.2', 'boto3', 'importlib2',
+                'fasteners']
+if platform.system() == 'Linux':
```

Comment:
Could you describe how you tested these changes? And on what systems.

Reply:
Tested the changes the following machines
1. Ubuntu server ec2 instance
2. AMI ec2 instance.
3. PureOS(Debian) system.
4. MacOS laptop
The installation works on all these machines.

However , due to ongoing ONNX issue, it is not working DLAMI.